pkg/envtest/server.go

```diff
@@ -46,9 +46,8 @@ func defaultAssetPath(binary string) string {
 
 }
 
-// APIServerDefaultArgs are flags necessary to bring up apiserver.
-// TODO: create test framework interface to append flag to default flags.
-var defaultKubeAPIServerFlags = []string{
+// DefaultKubeAPIServerFlags are default flags necessary to bring up apiserver.
+var DefaultKubeAPIServerFlags = []string{
```

Comment:
Is there a need to export this anymore given that we have made `KubeAPIServerFlag` configurable (change below) ?

Reply:
yes, because we still want to allow people to make use of the default args if they just want to append.